scripts/flattener.py

```diff
@@ -1647,8 +1648,10 @@ def main(mfinal_id):
 
 	# Printing out list of warnings
 	for n in warning_list:
-		print(n, end = '\n')
-		logging.warning(n)
+		if 'WARNING: Full list' not in n:
+			print(n, end = '\n')
+		if 'Full list available in logging file.' not in n:
```

Comment:
I believe `Full list available in logging file` refers to the warning that prints the first 10 drop_unmapped.

Reply:
Not sure how to do this, but line 960 has also logging that needs to not print to screen, and only to file:

for key in stats:
        stats[key] = set(stats[key])
	overlap_norm = set(mfinal_adata_genes).intersection(stats[key])
	warning_list.append("{}\t{}\t{}\t{}".format(key, len(stats[key]), len(overlap_norm), overlap_norm))